app/src/main/java/org/simple/clinic/summary/PatientSummaryModule.kt

```diff
@@ -2,10 +2,11 @@ package org.simple.clinic.summary
 
 import dagger.Module
 import dagger.Provides
+import org.threeten.bp.Duration
 
 @Module
 class PatientSummaryModule {
 
   @Provides
-  fun providesSummaryConfig() = PatientSummaryConfig(numberOfBpPlaceholders = 3)
+  fun providesSummaryConfig() = PatientSummaryConfig(numberOfBpPlaceholders = 3, bpEditableFor = Duration.ofMinutes(1L))
```

Comment:
The duration set for `bpEditableFor` should be configurable from server.
If we want to test on staging, we can change Date/Time settings on the phone.

Can we change the duration to the production value?